insights/specs/__init__.py

```diff
@@ -332,6 +332,7 @@ class Specs(SpecSet):
     ls_ocp_cni_openshift_sdn = RegistryPoint()
     ls_origin_local_volumes_pods = RegistryPoint()
     ls_osroot = RegistryPoint()
+    ls_rsyslog_errorfile = RegistryPoint(multi_output=True)
```

Comment:
It's not so good from the using perspective to mark it as `multi_output=True`, since it's from a `datasource` better change it to a single file.